### PR TITLE
LPS-173840 [EVP] Implementation for CSV download

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-173840](https://issues.liferay.com/browse/LPS-173840)

### CODE REVIEW

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by luisacmn** [comment]
Just starting reviewing :)

**Comment by EuniceTravassos** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
Just starting reviewing :)

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
It's necessary to add the:  `feature.flag.LPS-154672=true`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/routes/reports/generateReport/index.tsx [hunk 1]
```diff
@@ -119,8 +148,12 @@ const GenerateReport = () => {
 			return;
 		}
 
-		await downloadCsvFiltered();
-		await getRequestsByFilter(data).then((response) => response);
+		await getRequestsByFilter(data).then((response) => {
+			setTimeout(() => {
+				constructionFieldsCsv(response.items);
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/routes/reports/generateReport/index.tsx:153]:
```suggestion
				constructionFieldsCsv(response?.items);
```

**Review by eliasantosf — CHANGES_REQUESTED** [review]

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @juliocezar96

**Comment by eliasantosf** [comment]
ci:forward

**Pull request closed without merging**